mailpoet/readme.txt

```diff
@@ -105,39 +105,33 @@ Please note:
 
 = Translations =
 
-**Official translations**
-
+* Albanian
 * Arabic
 * Catalan
-* Chinese
 * Czech
 * Danish
 * Dutch
-* French (FR)
+* Dutch → Dutch (Formal)
+* French (Canada)
+* French (France)
 * German
+* German (Switzerland)
+* German → German (Formal)
```

Comment:
Is this intentional? Now we have 
* German
* German (Switzerland)
* German → German (Formal)

Shouldn't it only be 

* German (Formal)
* German (Switzerland)

? The same for Dutch

Reply:
Absolutely! 6695b4e912985150dd3f5a10a06e5780b9bbd856